applications/nrf_desktop/src/modules/ble_adv_ctrl.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define MODULE ble_adv_ctrl
+#include <caf/events/module_state_event.h>
+#include <caf/events/module_suspend_event.h>
+#include "usb_event.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_ADV_CTRL_LOG_LEVEL);
+
+static bool ble_adv_suspended;
```

Comment:
Consider limiting scope to `handle_usb_state_event` function. Alternatively we could keep here as bitmask (that might be easily expanded in the future)